_scripts/instruction-widget/templates/getting-started/renewal.html

```diff
@@ -37,4 +32,12 @@ <h4>Note:</h4>
 revocation happened for some reason). Please select a random minute within the
 hour for your renewal tasks.</p>
 </aside>
-{{/cron_included}}
+
+<p>An example cron job might look like this, which will run at noon and midnight every day:</p>
+
+<pre>0 0,12 * * * python -c 'import random; import time; time.sleep(random.random() * 60)' && {{base_command}} renew {{#cbauto}}--no-self-upgrade{{/cbauto}}</pre>
```

Comment:
This is my fault. I said we should use `random.random() * 60`, but this only spaces things out over a minute. We should do `random.random() * 3600` to spread it over the hour.